refactoring/token-replace.py

```diff
@@ -0,0 +1,77 @@
+#!/bin/env python3
+
+usageHelp = r"""
+
+Replace a given token string with another string in a file (but only touch the
+file if there were changes).
+
+This will only match complete tokens that match the regex:
+
+  ([^A-Za-z0-9_])|^)<token-to-replace)[^A-Za-z0-9_]
+"""
+
+
+def getCmndLineOptions():
+  from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+  clp = ArgumentParser(description=usageHelp,
+    formatter_class=RawDescriptionHelpFormatter)
+
+  clp.add_argument(
+    "-t", dest="tokenToReplace", required=True,
```

Comment:
May want to validate that this argument is a valid token (`^[a-zA-Z0-9_]+$`).

Reply:
If I add unit tests for these simple tools, I will keep that in mind.  One would always carefully examine the impact of running this tool to replace tokens by running `git diff` to make sure it did what you expect.